### PR TITLE
IBM Power update for Hosted Control Planes

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.14+

Requires CM

Issue: https://issues.redhat.com/browse/MULTIARCH-3703

Link to docs preview:
- https://68627--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/#hosted-control-planes-overview_hcp-overview
- https://68627--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-configuring#hcp-configuring-ibmpower

Related PR: https://github.com/openshift/openshift-docs/pull/68637

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by manojnkumar** [comment]
LGTM

**Review by mkumatag — APPROVED** [review]
/lgtm

File: hosted_control_planes/hcp-configuring.adoc
[the remark this answers is not in the document]
Comment (alishaIBM) [reply hosted_control_planes/hcp-configuring.adoc:1]: Hi @lahinson , thank you for the review. I have removed the updates from `hcp-configuring.adoc` from the PR.

**Review by lahinson — APPROVED** [review]
LGTM. Thank you, @alishaIBM!

**Comment by alishaIBM** [comment]
PTAL: @sferich888, @davemulford, @xltian, @kalexand-rh, @sjstout

**Comment by davemulford** [comment]
LGTM, thanks!

**Comment by kalexand-rh** [comment]
LGTM

**Comment by xenolinux** [comment]
LGTM! Can you please update the preview link? I think the correct one is https://68627--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/#hosted-control-planes-overview_hcp-overview

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

**Comment by alishaIBM** [comment]
> > LGTM! Can you please update the preview link? I think the correct one is https://68627--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/#hosted-control-planes-overview_hcp-overview
> > I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!
> 
> Peer review LGTM! @alishaIBM Correct me if I'm wrong, but I noticed that you have specified the version as `4.14`, but seeing that you have branched off `main`, I am assuming that these are for 4.14+ (i.e `main`, `4.14`, and `4.15`). If that's the case, pls update the version accordingly. Thanks!

@abhatt-rh I have updated the version. Please check. Thanks.

**Comment by abhatt-rh** [comment]
> > > LGTM! Can you please update the preview link? I think the correct one is https://68627--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/#hosted-control-planes-overview_hcp-overview
> > > I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!
> > 
> > 
> > Peer review LGTM! @alishaIBM Correct me if I'm wrong, but I noticed that you have specified the version as `4.14`, but seeing that you have branched off `main`, I am assuming that these are for 4.14+ (i.e `main`, `4.14`, and `4.15`). If that's the case, pls update the version accordingly. Thanks!
> 
> @abhatt-rh I have updated the version. Please check. Thanks.

Thanks, @alishaIBM. I have now added the relevant labels and milestones.

**Pull request merged**